projectroles/models.py

```diff
@@ -820,6 +820,24 @@ def get_setting_value(
         if not app_name == 'projectroles':
             query_parameters['app_plugin__name'] = app_name
         setting = super().get_queryset().get(**query_parameters)
+        # plugin = get_app_plugin(app_name)
```

Comment:
The validation is already done when we call this in `AppSettingAPI`, so no further checks should be needed here? Removing the check for project and user above is correct.